protocol/0062-SPAM-spam_protection.md

```diff
@@ -80,6 +80,22 @@ The minimum allowed withdrawal amount is `spam.protection.minimumWithdrawalQuant
 
 Any withdrawal request for a smaller amount is immediately rejected.
 
+### Referral spam
+
+The [on-chain referral program](./0083-RFPR-on_chain_referral_program.md) adds three transaction types which can be submitted with no cost/risk to the party:
+
+- `CreateReferralSet`
+- `UpdateReferralSet`
+- `ApplyReferralCode`
+
+To avoid spamming of `CreateReferralSet` and `UpdateReferralSet` transactions, a party must meet the staked governance tokens ($VEGA) threshold set by the network parameter `referralProgram.minStakedVegaTokens`. A party who does not meet this requirement should have any transactions of the aforementioned types pre-block rejected.
+
+To avoid spamming of `ApplyReferralCode`, a party must meet the deposited funds threshold set by the network parameter `spam.protection.applyReferral.min.funds`.  All non-VEGA assets count towards this threshold and balances should be scaled appropriately by the assets quantum. A party who does not meet this requirement should have any transactions of the aforementioned type pre-block rejected. This requirement will be checked against snapshots of account balances taken at a frequency determined by the network parameter `spam.protection.balanceSnapshotFrequency`. This network parameter is a duration (e.g. `5s`, `1m5s`).
```

Comment:
question: why exclude vega? i.e. suppose I have a 1000 VEGA in my general account that I received in rewards, why not let it count towards the min funds? (as opposed to staking which is not included in this)

Reply:
The thinking was this threshold is specified as a quantum and $VEGA is fairly volatile in comparison to the current settlement assets on mainnet (so the assets quantum might not be accurate). The reason it is singled out specifically is because it is given out as a reward so will likely be in a lot of parties wallets.

I'm not attached to this restriction though, and thinking about it more it will just seem weird if more volatile assets do get proposed later down the line.